test/specs/custom-bundling-roots/reference/openapi-3.bundled.js

```diff
@@ -14,10 +14,7 @@ module.exports = {
     "/flight/{id}": {
       parameters: [
         {
-          type: "number",
-          name: "id",
-          in: "path",
-          required: true
+          $ref: "#/components/schemas/Id"
```

Comment:
IIRC we didn't want to have a $ref here, but I don't remember why?
Or am I wrong?
TBH, I don't see a reason not to have a $ref here if it was authored in the original source.

Reply:
I think it was our oversight in the end.